reviews-receiver/src/routes/api-key.router.ts

```diff
@@ -0,0 +1,59 @@
+import { Router } from 'express'
+import ApiKey from '../models/api-key.model'
+import crypto from 'crypto'
+import { Request, Response } from 'express'
+
+const apiKeyRouter = Router()
+
+/**
+ * @swagger
+ * /keys:
+ *   post:
+ *     tags:
+ *       - Keys
+ *     summary: Generate a new API key
+ *     requestBody:
+ *       required: true
+ *       content:
+ *        application/json:
+ *         schema:
+ *          type: object
+ *          properties:
+ *           businessId:
+ *            type: string
+ *            description: The ID of the business
+ *          example:
+ *           businessId: 'asdasdm324jnr2kjwnfsdd'
+ *
+ *     responses:
+ *       201:
+ *         description: Successful operation
+ *       400:
+ *        description: Bad request
+ *       500:
+ *        description: Internal server error
+ *
+ */
+
+// TODO: set up the route to generate a new API key in seperate service
```

Comment:
isn't this a route that generates a new api key?

Reply:
yes it is
it should be in another service
in the current situation use can issue key for himself..